feg/cloud/configs/service_registry.yml

```diff
@@ -43,9 +43,9 @@ services:
       feg_hello:
         port: 9103
       ocs:
-        port: 9079
+        port: 9103
```

Comment:
I think, the change should be done for session_proxy (LN: 39 above) not ocs & pcrf here